decentraland/unity-explorer
# feat: Show / Hide UI pressing U

## What does this PR change?
Fix #605
Fix #590 

When we press the U key, the UI is hidden or unhidden.

## How to test the changes?
1. Launch the explorer.
2. Press [U] key.
3. Observe all the UIs dissapear in the screen: Normal UIs, Debug Panel UI, Scene's UI and Cursor UI.
4. Press [U] key again.
5. Observe all the UIs appear again in the screen.

## Our Code Review Standards
https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
